style.css

```diff
@@ -2,110 +2,566 @@ html,
+  background: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  display: inline-block;
   text-decoration: none;
+  position: relative;
+  border: 1px solid #6070ff;
+  order: 3;
+  transition-duration: 0.4s;
+  margin-left: 47px;
+  margin-top: 27px;
+}
+
+.submit:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+  color: white;
+}
+
+.submit:active {
+  background: #2230d2;
+  color: white;
+}
+
+.submit:enabled {
+  border: 1px solid #7f8cff;
+  border-radius: 8px;
+}
+
+.submit:disabled {
+  background: #e5e5e5;
+}
+
+.bottom-sec {
+  display: grid;
+  grid-auto-rows: 200px;
+  grid-auto-columns: 100%;
+  height: 1483px;
+  width: 100%;
+}
+
+.section-bottom {
+  grid-row-start: 1;
+  grid-row-end: 3;
+  grid-column-start: 1;
+  background-color: #fff;
+  height: 924px;
+  display: flex;
+  width: 100%;
+  border-top-right-radius: 60px;
+}
+
+.contact-us {
+  grid-row-start: 4;
+  grid-row-end: 9;
+  grid-column-start: 1;
+  z-index: 10;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: url(images/background-contact.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 846px;
+  border-top-left-radius: 90px;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding-left: 30rem;
+  padding-right: 30rem;
+}
+
+.contact-form input {
+  height: 48px;
+  width: 447px;
+  left: 0;
+  top: 0;
+  margin-left: 24px;
+  margin-right: 24px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  padding-left: 16px;
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  font-size: 17px;
+  color: #172b4d;
+}
+
+.contact-form input::placeholder {
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  font-size: 17px;
+  padding-left: 16px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  color: #172b4d;
+}
+
+.contact-form textarea {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  padding-left: 16px;
+  padding-top: 15px;
+  height: 170px;
+  width: 447px;
+  margin-left: 24px;
+  margin-right: 24px;
+  bottom: 0;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+.contact-form textarea::placeholder {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  padding-left: 16px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+}
+
+.contact-form button {
+  inline-size: 0;
+  padding: 12px 16px;
+  width: 167px;
+  height: 58px;
+  margin-left: 139px;
+  margin-bottom: 35px;
+  background: #fff;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.bottom-sec .section-bottom .right {
+  width: 60%;
+  padding: 3rem auto 0 5rem;
+}
```

Comment:
- [ ] I have noticed that the languages are not responsive to different desktop screen sizes. I suggest that you consider, fixing the responsiveness such that your design matches the Figma template design ☺️.
- [ ] Please make sure that your mobile design still looks the same way it was.
Your design 😎 
![image](https://github.com/ethred/Portfolio/assets/99394374/cf5eba3e-8c43-41b6-aa21-02ef47fdce1a)